djoser/views.py

```diff
@@ -130,11 +130,12 @@ def _action(self, serializer):
     def get_users(self, email):
         if self._users is None:
             email_field_name = get_user_email_field_name(User)
-            active_users_kwargs = {
-                email_field_name + '__iexact': email, 'is_active': True
+            filter_users_kwargs = {
+                email_field_name + '__iexact': email
             }
```

Comment:
Yup this needed a line break, unless `email_field_name` is renamed to something <= 5 characters to fit in one line